tests/s3/test_s3.py

```diff
@@ -0,0 +1,12 @@
+from time import sleep 
+
+def test_localstack_is_healthy(get_docker_api_client):
+
+    api_client = get_docker_api_client
```

Comment:
why are we testing localstack?

Reply:
I think it provides added reassurance in the services we use in our testing. we already include a healthcheck on it in the compose file though which could make this test redundant.

Reply:
removed localstack test [here](https://github.com/GSA/datagov-harvesting-logic/pull/10/commits/78e7320f86505509fa384cd30694f556f56d377f)